yarn.lock

```diff
@@ -2,6 +2,28 @@
 # yarn lockfile v1
 
 
+"@actions/core@^1.2.4":
```

Comment:
The `yarn.lock` diff is from the Chromatic update in `master`. Not sure why the base comparison is so imprecise with these branches.